README.md

```diff
@@ -1,12 +1,12 @@
 SonarLint Core
 ==============
-Core library to run SonarLint analysis (used by SonarLint Eclipse and IntelliJ), as well as the SonarLint language server (used by SonarLint VSCode)
+Core library to run SonarLint analysis (used by SonarLint Eclipse and IntelliJ), as well as the SonarLint language server (used by SonarLint VSCode) with the goal of helping developers code [Clean Code](https://www.sonarsource.com/solutions/clean-code/?utm_medium=referral&utm_source=github&utm_campaign=clean-code&utm_content=sonarlint-core).
```

Comment:
I found the "helping developers code Clean Code" wording a bit odd, and I double checked with Jonathan, we agreed to reuse the wording used in sonarlint-eclipse:
> Core library to run SonarLint analysis (used by SonarLint Eclipse and IntelliJ), as well as the SonarLint language server (used by SonarLint VSCode). SonarLint helps you deliver [Clean Code](https://www.sonarsource.com/solutions/clean-code/?utm_medium=referral&utm_source=github&utm_campaign=clean-code&utm_content=sonarlint-core) in your IDE.

I can make the change tomorrow

Reply:
Great, thanks for your help